src/screens/discover/DiscoverDetailScreen.tsx

```diff
@@ -71,34 +61,76 @@ const requestHandler = async (props: RequestHandlerProps) => {
 };
 
 const DiscoverDetailScreen: React.FC = () => {
+    const query: string = useLoaderData() as string;
     const { profile, setProfile } = useProfileContext();
     const path = window.location.pathname
         .match(/\/\w+$/)
         ?.join()
         .slice(1);
-    const [data, setData] = useState<ShowData[] | null>(null);
     const [loading, setLoading] = useState<boolean>(true);
+    const storageItem = localStorage.getItem('streamabilityView');
```

Comment:
It's worth noting that this will be shared between discover details and search results. Meaning if you toggle the view on search results, it would also update how discover details is displayed. Not entirely sure if this is a problem or not. But we could use a unique key to avoid this.

Reply:
separated the local storage keys for results and discover details screens.